src/index.tsx

```diff
@@ -9,7 +9,7 @@ class TheButton extends React.Component<IButtonProps> {
         super(props);
     }
 
-    public render() {
+    render() {
```

Comment:
`public` ist in Typescript default, da es in Javascript nur public gibt und valides Javascript auch valides Typescript ist

Reply:
Nur mag der Linter das leider nicht... 🤔Regel abschalten?

![image](https://user-images.githubusercontent.com/4947715/37852948-d73bc610-2ee4-11e8-9413-f84e774c7d42.png)

Reply:
Habe ich nicht gesehen - ruhig drin lassen. Wird schon seinen Grund haben 🤔
Komisch das VS Code das nicht anzeigt...

Reply:
Hat es bei mir auch erst nicht getan... Jetzt plötzlich doch. Boot tut goot.

Reply:
Es wäre aber gut, wenn VS Code und `npm run lint` die gleichen Findings anzeigen würden. Ob das ein Fehler im VS Code TSLint Plugin ist?

Reply:
Bei mir sind die beiden wie erwähnt synchron...

![image](https://user-images.githubusercontent.com/4947715/37853675-01df39e4-2ee8-11e8-8273-5bf1d3e2dfdf.png)

Reply:
klappt bei mir jetzt auch